src/rest/Facet/FacetSearchRequest.ts

```diff
@@ -1,5 +1,16 @@
 import { IQuery } from '../Query';
 
+export enum FacetSearchType {
```

Comment:
The documentation here is copied from the model in the [Swagger documentation](https://platformdev.cloud.coveo.com/docs/?api=SearchApi#!/Search/post_rest_search_v2_facet).